src/plugins/data/public/index.ts

```diff
@@ -26,27 +26,27 @@ import { PluginInitializerContext } from '../../../core/public';
  */
 
 import {
-  FILTERS,
   buildEmptyFilter,
-  buildPhrasesFilter,
   buildExistsFilter,
   buildPhraseFilter,
+  buildPhrasesFilter,
   buildQueryFilter,
   buildRangeFilter,
-  toggleFilterNegated,
   disableFilter,
+  FILTERS,
   FilterStateStore,
+  getDisplayValueFromFilter,
   getPhraseFilterField,
   getPhraseFilterValue,
-  isPhraseFilter,
   isExistsFilter,
-  isPhrasesFilter,
-  isRangeFilter,
+  isFilterPinned,
   isMatchAllFilter,
   isMissingFilter,
+  isPhraseFilter,
+  isPhrasesFilter,
   isQueryStringFilter,
-  getDisplayValueFromFilter,
-  isFilterPinned,
+  isRangeFilter,
+  toggleFilterNegated,
```

Comment:
Did some alphabetizing a few places in the index file; sorry if the diffs are hard to follow 😉